data/report_templates/project_summary.md

```diff
@@ -190,6 +190,17 @@ Generation sequence Cluster Storage, [uppmax.uu.se](http://www.uppmax.uu.se)) ca
 - [NGI data delivery note](https://ngisweden.scilifelab.se/info/Data%20delivery)
 - [UPPMAX GRUS user guide](https://www.uppmax.uu.se/support/user-guides/grus-user-guide/)
 
+{% elif project.cluster == 'dds' -%}
+## Data access at DDS
+
+Data from the sequencing have been be uploaded to the **Data Delivery System**
```

Comment:
```suggestion
Data from the sequencing have been uploaded to the **Data Delivery System**
```

Reply:
I guess the text for grus should be updated as well?

Reply:
Sorry I just copied without reading :D